### DISCUSSION TITLE
Duplicating a section of a table after adding a Repeater

### DISCUSSION THREAD

**Discussion by KevMyst** [Budibase/budibase#3952] [Help]
Duplicating a section of a table after adding a Repeater

![duplicating a section of a table after adding a Repeater](https://user-images.githubusercontent.com/6716949/148428920-c8b7e48f-8072-454c-a44c-ad2d4883a882.png)

**Comment by mike12345567** [comment]
Hi @KevMyst - there may be some confusion here, with your current component tree setup this is the intended behaviour.

You have two rows in your table, which are coming from the `Data Provider`. The repeater will duplicate all the components within it for every row that the data provider retrieves - in this case that means creating the table twice.
 
If you move the table and its filter to be under the data provider, rather than under the repeater then you will only have one table shown.

**Reply by KevMyst** [reply]
Thanks for the quick reply.
I used the repeater to translate the text when there are no rows;)
Is there any other way to translate?
![No rows found](https://user-images.githubusercontent.com/6716949/148430041-cf6e0563-9317-4a8e-b0a5-dc2e12312f46.png)

**Reply by mike12345567** [reply]
Hi @KevMyst currently elements like that cannot be translated, this is something we plan to work on but I don't have any idea of timeline right now!
